### PR TITLE
Proposed New Wyvern Rider Description

### PR DESCRIPTION
**Pull request by Gothyoba**
I know I already made a PR for this as well as some other descriptions around a year ago, but the writing was not very good and I ultimately abandoned the idea. However, I have decided to this to propose a new and hopefully better description.

### CODE REVIEW

**Comment by Wedge009** [comment]
I'm guessing English is not your native language. There's a bit of inconsistency with 'travellers' using en_GB spelling while 'skillfully' is en_US spelling. Wesnoth uses en_US by default so should probably change 'travellers' to 'travelers'.

It's not a bad description (and I'm a bit surprised the current description is blank), but it doesn't quite flow smoothly to me as a native Anglophone. But then, I'm also not a writer. Probably best for this to be reviewed by those in charge of the prose text.

File: data/core/units/dunefolk/Wyvern_Rider.cfg [hunk 1]
```diff
@@ -14,7 +14,7 @@
     {AMLA_DEFAULT}
     cost=102
     usage=scout
-    description=""
+    description= _ "Wyverns are fierce and angry beasts, especially when provoked. Most travellers would run or fight in their mighty presence. Yet a few are able to tame these wild creatures. These dunefolk riders spend years, even decades of their lives bonding with creatures. They skillfully dedicate their lives to art of flying the Wyvern. While these masters do not usually use this art in warfare, they have been known to become threatening warriors when defending their home."
```
Comment (CelticMinstrel) [inline data/core/units/dunefolk/Wyvern_Rider.cfg:17]: Preliminary suggested changes:

* Merge sentences 2 and 3 into a single sentence, with a comma
* Add an article for "creatures"
* Make "home" plural
* Don't capitalize the race name

The result:
```suggestion
    description= _ "Wyverns are fierce and angry beasts, especially when provoked. Most travellers would run or fight in their mighty presence, yet a few are able to tame these wild creatures. These dunefolk riders spend years, even decades of their lives bonding with the creatures. They skillfully dedicate their lives to art of flying the wyvern. While these masters do not usually use this art in warfare, they have been known to become threatening warriors when defending their homes."
```

I think it still has some problems even with these changes. In particular, the sentence with "skillfully" feels awfully abrupt.

**Comment by CelticMinstrel** [comment]
Still a bit unsatisfying, mainly the "greatly linked" and the following sentence…

**Comment by Gothyoba** [comment]
I changed some parts of the description. I think it sounds better now, but I’m not quite happy with it, especially at the end. If this does not get added, I think the Wild Wyvern description should probably be used here as a placeholder. As far as I’m aware, this is the only completely blank description.

**Comment by CelticMinstrel** [comment]
Okay, at this point I don't have anything else to add, so I say we should wait and see what @nemaara things.

**Review by knyghtmare — CHANGES_REQUESTED** [review]
Maybe add these changes:

```txt
Although generally peaceful, when provoked wyverns can easily become fierce and angry beasts. When truly enraged they can even tear apart experienced warriors with relative ease. Most travelers would run or fight in their mighty presence, yet a few are able to tame these wild creatures. These Dunefolk riders spend years, even decades of their lives bonding with them, slowly building up their trust until they have gained true mastery over the skies. The most experienced of these Wyvern riders have ascended to be the greatest and most revered travelers of the land. While they do not usually use their knowledge for warfare, they can be threatening warriors when defending their homes.
```

**Comment by Dalas121** [comment]
Is there any established lore on Wyvern Riders and how (if?) they fit into canon? Any high-profile UMC that feature them? Or are we working from a blank slate.

**Comment by Gothyoba** [comment]
@knyghtmare This looks good to me. I’ve added the changes.
@Dalas121 I don’t know of any UMC that use the Wyvern Rider in any significant way. I personally haven’t found any but I’m probably wrong given the amount of dunefolk campaigns.

**Comment by CelticMinstrel** [comment]
Didn't they originally come from a user-made campaign? I think it might've been Oath of Allegiance. It does have a wyvern rider line, at least, not sure if it's the _original_ source though.

**Review by Dalas121 — APPROVED** [review]

**Comment by CelticMinstrel** [comment]
The full sentence:

> Although generally peaceful, when provoked wyverns can easily become fierce and angry beasts.

Putting a semicolon there doesn't make any sense. The proposed comma would be acceptable, but I think it flows fine without it. It would flow even better if "when provoked" moved to the end of the sentence though… but maybe there was a reason for fronting it?

**Comment by Wedge009** [comment]
> Although generally peaceful, when provoked wyverns can easily become fierce and angry beasts.

I think the punctuation is okay here.

**Review by knyghtmare — APPROVED** [review]

**Pull request merged**